Sources/StreamChat/Database/StreamChatModel.xcdatamodeld/StreamChatModel.xcdatamodel/contents

```diff
@@ -133,6 +134,7 @@
         <relationship name="channelConfig" optional="YES" maxCount="1" deletionRule="Nullify" destinationEntity="ChannelConfigDTO" inverseName="commands" inverseEntity="ChannelConfigDTO"/>
     </entity>
     <entity name="CurrentUserDTO" representedClassName="CurrentUserDTO" syncable="YES">
+        <attribute name="blockedUserIds" optional="YES" attributeType="Transformable"/>
```

Comment:
Out other transformable types have value transformer set to `NSSecureUnarchiveFromData`